WPFWijkagent/WPFWijkagent/App.xaml.cs

```diff
@@ -7,5 +9,60 @@ namespace WijkagentWPF
     /// </summary>
     public partial class App : Application
     {
+        public const string SessionFile = "./Session.cfg";
```

Comment:
even een vraag. kan dit niet gewoon met de app.config??? krijgen we anders geen problemen met rechten om bestanden aan te maken?